### ISSUE TITLE
failed to extract archive download from … with 1.6.0-rc2 in Windows 10

### ISSUE THREAD

**Issue by HaoxuanGuo** [JuliaLang/julia#40040]
```julia
julia> versioninfo()
Julia Version 1.6.0-rc2
Commit 4b6b9fe4d7 (2021-03-11 07:05 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: AMD Ryzen 9 5950X 16-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, generic)
```

Just add any a Package and gets this error. Here use CUDA.jl for sample:

```julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.6.0-rc2 (2021-03-11)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.6) pkg> add CUDA
  Installing known registries into `C:\Users\skyair\.julia`
     Cloning registry from "https://github.com/JuliaRegistries/General.git"
       Added registry `General` to `C:\Users\skyair\.julia\registries\General`
   Resolving package versions...
┌ Warning: failed to extract archive downloaded from https://pkg.julialang.org/package/c03570c3-d221-55d1-a50c-7939bbd78826/2b1dfcba103de714d31c033b5dacc2e4a12c7caa
└ @ Pkg.Operations C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\Pkg\src\Operations.jl:590
┌ Warning: failed to extract archive downloaded from https://pkg.julialang.org/package/e2ba6199-217a-4e67-a87a-7c52f15ade04/10407a39b87f29d47ebaca8edbc75d7c302ff93e
└ @ Pkg.Operations C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\Pkg\src\Operations.jl:590
┌ Warning: failed to extract archive downloaded from https://pkg.julialang.org/package/79e6a3ab-5dfb-504d-930d-738a2a938a0e/ffcfa2d345aaee0ef3d8346a073d5dd03c983ebe
└ @ Pkg.Operations C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\Pkg\src\Operations.jl:590
┌ Warning: failed to extract archive downloaded from https://pkg.julialang.org/package/a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f/32cdbe6cd2d214c25a0b88f985c9e0092877c236
└ @ Pkg.Operations C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\Pkg\src\Operations.jl:590
┌ Warning: failed to extract archive downloaded from https://pkg.julialang.org/package/6c6a2e73-6563-6170-7368-637461726353/ad4b278adb62d185bbcb6864dc24959ab0627bf6
└ @ Pkg.Operations C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\Pkg\src\Operations.jl:590
...
     Cloning [a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f] TimerOutputs from https://github.com/KristofferC/TimerOutputs.jl.git
   Installed TimerOutputs ─────── v0.5.8
     Cloning [6c6a2e73-6563-6170-7368-637461726353] Scratch from https://github.com/JuliaPackaging/Scratch.jl.git
   Installed Scratch ──────────── v1.0.3
     Cloning [79e6a3ab-5dfb-504d-930d-738a2a938a0e] Adapt from https://github.com/JuliaGPU/Adapt.jl.git
   Installed Adapt ────────────── v3.2.0
     Cloning [c03570c3-d221-55d1-a50c-7939bbd78826] Memoize from https://github.com/JuliaCollections/Memoize.jl.git
 ...
     Cloning [052768ef-5323-5732-b1bb-66c8b64840ba] CUDA from https://github.com/JuliaGPU/CUDA.jl.git
   Installed CUDA ─────────────── v2.6.1
    Updating `C:\Users\skyair\.julia\environments\v1.6\Project.toml`
  [052768ef] + CUDA v2.6.1
    Updating `C:\Users\skyair\.julia\environments\v1.6\Manifest.toml`
  [621f4979] + AbstractFFTs v1.0.1
  [79e6a3ab] + Adapt v3.2.0
  [ab4f0b2a] + BFloat16s v0.1.0
...
  [8e850ede] + nghttp2_jll
  [3f19e933] + p7zip_jll
  Progress [========================================>]  24/24
24 dependencies successfully precompiled in 30 seconds

(@v1.6) pkg>
```

**Comment by KristofferC** [comment]
As posted in https://github.com/JuliaLang/julia/issues/40021#issuecomment-799345418, the error is:

`ERROR: schannel: failed to receive handshake, SSL/TLS connection failed while requesting https://github.com/JuliaBinaryWrappers/Rmath_jll.jl/releases/download/Rmath-v0.2.2+1/Rmath.v0.2.2.x86_64-w64-mingw32.tar.gz`

cc @StefanKarpinski

**Comment by StefanKarpinski** [comment]
Possible firewall issue?

**Comment by HaoxuanGuo** [comment]
> Possible firewall issue?

I think may not. Because it can be downloaded by my browser (Microsoft Edge (Chromium Edition)).

@StefanKarpinski

**Comment by HaoxuanGuo** [comment]
It has been fixed in 1.6.0. Thanks.
